src/diffusers/pipelines/stable_diffusion/pipeline_flax_stable_diffusion_img2img.py

```diff
@@ -224,7 +224,8 @@ def _generate(
         # Create init_latents
         init_latent_dist = self.vae.apply({"params": params["vae"]}, image, method=self.vae.encode).latent_dist
         init_latents = init_latent_dist.sample(key=prng_seed).transpose((0, 3, 1, 2))
-        init_latents = 0.18215 * init_latents
+        init_latents = self.vae.config.scaling_factor * init_latents
+        latents_shape = (batch_size, self.unet.in_channels, height // 8, width // 8)
```

Comment:
Why is this shape computed here again? It was already used in lines 212 to 223.

Reply:
Good catch, maybe got leftover from merge conflicts.